sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -401,26 +400,199 @@ const documents = [
 ];
 
 async function main() {
-  const poller = await client.beginAnalyzeHealthcareEntities(documents);
+  const poller = await client.beginAnalyzeBatch(
+    [
+      {
```

Comment:
would it be helpful to pull the configuration object out into a separate variable? Or would that defeat the purpose of letting TS narrow things inline?

Reply:
Narrowing is important here because the list of parameters supported for each action depends on what value you put in `kind`.

Reply:
I figured :)

Reply:
The narrowing may work if we type the variable? `const action: AnalyzeBatchAction = { kind: "..." }`?  If yes it is probably easier to read when there are multiple actions.

Reply:
Good point, addressed.